### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by OlesyaGerasimenko**

### CODE REVIEW

File: templates/ru/choose-model.tmpl [hunk 1]
```diff
@@ -48,20 +31,16 @@ function filterItems(el) {
 {?printer_is_shared=?Нет совместного доступа:{?printer_is_shared=0?Нет совместного доступа:Разрешить совместный доступ}} к этому принтеру</TD>
 </TR>
 <TR>
-<TH CLASS="label">Производитель:</TH>
-<TD>{PPD_MAKE} <INPUT TYPE="SUBMIT" NAME="SELECT_MAKE" VALUE="Выбрать другого производителя"></TD>
-</TR>
-<TR>
```
Comment (michaelrsweet) [inline templates/ru/choose-model.tmpl:54]: Looks like this removes a previous change for the driver filtering feature? Please rebase and update this PR.

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
Please correct the filtering change, then I think it is good to go...

File: templates/ru/choose-model.tmpl (hunk 1 above)
Comment (OlesyaGerasimenko) [reply templates/ru/choose-model.tmpl:54]: I've fixed the text in the file -- could you please help me out with rebasing and updating the PR? I'm not sure I can do it correctly.
[choose-model.zip](https://github.com/OpenPrinting/cups/files/13794168/choose-model.zip)

Comment (michaelrsweet) [reply templates/ru/choose-model.tmpl:54]: I think you just need to push the change to your branch, but if that isn't enough you can just create a new PR once the change is pushed.

Comment (OlesyaGerasimenko) [reply templates/ru/choose-model.tmpl:54]: I've just updated the file in my branch, is that it?

Comment (michaelrsweet) [reply templates/ru/choose-model.tmpl:54]: Doesn't look like Github picked it up. Go ahead and just create another pull request...

**Pull request closed without merging**